### ISSUE TITLE
Add more Lyrics Sources API - Multi-Lingual English Hindi

### ISSUE THREAD

**Issue by sammyviv** [ddddxxx/LyricsX#385]
Hi, Most lyrics fetched by the current sources are from Chinese based websites so even english lyrics sometimes have Chinese Artifacts or Translations. 

So fetching lyrics from API's which have a large multi lingual collection of English and Hindi lyrics would be much appreciated. If APIs aren't available for a particular website then web-scraping is quite efficient for getting the lyrics.

A list of freely available APIs and Websites:-
docs.genius.com
ytvdgrabber.com
metrolyrics.com
hindilyrics.net
lyricsmania.com
lyricsmasti.com
azlyrics.com

I really wouldn't even mind buying this app if it has such functionality. If you need any help testing it do let me know.

**Comment by ddddxxx** [comment]
New lyrics source is always welcome. However, currently LyricsX only accept time synced lyrics. I can't really make use of these plain text lyrics sources.

**Comment by sammyviv** [comment]
Oh right!! That makes complete sense... I'll try looking for synchronized lyrics sources.

I did find a few:-
Letras.mus.br (this one is pretty good)
megalobiz.com (any response from the owner for the API? 😋)
Lyrics Wikia (not 100% sure about this one though)

**Comment by ddddxxx** [comment]
Letras.mus.br looks great. They don't show time tag on their website thus web-scraping is not sufficient.

I'm curious how they synchronize lyrics. When I click "send lyrics" they don't ask me to synchronize lyrics. When I click "correct lyrics" I don't see any time tag at all. It loos like these plain text lyrics magically get time synced.

**Comment by sammyviv** [comment]
Yeah I had a look as well and you're right. Letras seems to handle time tags on their backend only accessible for their official apps, it might be possible to capture network traffic of their app and figure out the GET requests but it's a long shot. I might give it a try when I get some time. 

I do understand the time-sync concept of LyricX but for Lyrics which are simply not available on time-synced sources, it makes sense to implement a web-scraper which just displays plain text lyrics (in the window) and scrolls it according to time/length of the song (not accurate but acceptable).

**Comment by kahboom** [comment]
Bumping because this would be such a cool feature. Or at least some way for users to add their own lyrics with timestamps?

**Comment by akulgoel96** [comment]
@ddddxxx any update from megalobiz.com owner about the API?
